Source/Core/DolphinQt/Achievements/AchievementHeaderWidget.cpp

```diff
@@ -157,6 +160,13 @@ void AchievementHeaderWidget::UpdateData()
     m_game_progress_hard->setVisible(false);
     m_game_progress_soft->setVisible(false);
     m_rich_presence->setVisible(false);
+    if (AchievementManager::GetInstance()->IsCompromised())
+    {
+      m_locked_warning->setText(
+          tr("Achievements have been locked due to the player switching roms or otherwise "
+             "manipulating their boot.<br>Please close all running games to unlock achievements."));
```

Comment:
1. We normally don't use the term ROMs in Dolphin. (It also isn't capitalized correctly here.)
2. We normally address the user as "you".
3. "Manipulating [the] boot" is hard to understand the meaning of.